pydy/system.py

```diff
@@ -429,9 +417,8 @@ def _args_for_gen_ode_func(self):
 
         """
 
-        args = (self.eom_method.forcing_full,
-                self.coordinates,
-                self.speeds,
+        args = (self.force,
```

Comment:
I think right hand side is a good start. I'll check a text book to see if there are some better names.